picture-stealer/src/main/resources/db/migration/V3__create_indexes.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX IF NOT EXISTS camera_nasa_id_index ON camera (nasa_id);
```

Comment:
Hi! Why is there a need to create index manually if according to Postgres documentation: "Adding a unique constraint will automatically create a unique B-tree index on the column or group of columns listed in the constraint" ?

Reply:
It's not needed here, I forgot,  that field unique